.gitignore

```diff
@@ -16,6 +16,8 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.magic/
```

Comment:
**Suggestion** Can you pull this into a separate PR that I can instant approve/merge, please?  Otherwise, when I sync this PR internally, since `.gitignore` is not managed by Copybara, it will get undone during then nightly export.

Reply:
done. see #3531